data/transport missions.txt

```diff
@@ -1975,3 +1975,40 @@ ship "Science Drone"
 		"heat dissipation" 1
 	explode "tiny explosion" 5
 	explode "small explosion" 2
+
+
+
+mission "Loan Broker Reunion"
+	name "The Broker"
+	description "Transport the broker's enforcers to <destination> by <day>."
+	blocked "You'll need more room to house the hired thugs. Return when you have <capacity>."
+	deadline
+	cargo "Living necessities" 5
+	passengers 15
+	minor
+	to offer
+		not "year" == 3013
+	source "New Boston"
+	destination
+		government "Syndicate"
+	on offer
+		conversation
+			'As you're walking through the spaceport searching for a bar or pub, someone catches your eye. A nearly bald, middle-aged man wanders through the crowd in an expensively tailored suit, searching the faces that pass by him. You stop for a moment and look closer; he seems familiar in some way. Eventually, his gaze settles on you, making eye contact. He blinks a few times in surprise, then begins moving through the crowd towards you.'
```

Comment:
These should be backticks (`), not single quotes (').

`someone catches your eye`
I just don't like the use of `someone` here for some reason.

`You stop for a moment and look closer; he seems familiar in some way.`
-> `You pause and look at him more closely; he seems oddly familar.`